src/app/components/shell/header/header.scss

```diff
@@ -44,182 +74,583 @@
         }
     }
 
-    .secondary-nav {
-        display: flex;
-        justify-content: flex-end;
-        padding-right: 8vw;
+    //Generic styles
+    .container {
+      display: inherit;
+      margin: 0 auto;
+      padding: 0 1.5rem;
+      max-width: $boxed-width;
+      flex-flow: row wrap;
+      flex: 1 100%;
+    }
 
-        @at-root .collapsed#{&} {
-            display: none;
+    .nav-menu {
+      @include small {
+        opacity: 0;
+        @at-root .active#{&} {
+          display: flex;
+          opacity: 1;
+          transition: opacity 0.5s 0.7s;
+          flex-flow: column nowrap;
         }
+      }
+      .nav-menu-item {
+        &.parent > a {
+          position: relative;
+          &::before,
+          &::after {
+            border-right: 2px solid;
+            content: '';
+            display: block;
+            height: 10px;
+            margin-top: -7px;
```

Comment:
Avoid using `px`.  You can use `rem` instead (10px == 1rem, so it's an easy conversion).  `rem` is more flexible and allows for better support for accessibility.